ciao-deploy/deploy/setup.go

```diff
@@ -78,8 +79,13 @@ func createConfigurationFile(ctx context.Context, clusterConf *ClusterConfigurat
 
 	config.Configure.Launcher.ComputeNetwork = []string{clusterConf.ComputeNet}
 	config.Configure.Launcher.ManagementNetwork = []string{clusterConf.MgmtNet}
+
+	// Disk limit checking is broken and should always be disabled.
+	// See issue #1541
 	config.Configure.Launcher.DiskLimit = false
-	config.Configure.Launcher.MemoryLimit = false
+	if clusterConf.DisableLimits {
```

Comment:
I can't see how this has any effect.  By default,

config.Configure.Launcher.MemoryLimit = false

When we supply --disable-limits it will still be false.

Should this be
```
if !clusterConf.DisableLimits {
    config.Configure.Launcher.MemoryLimit = true
}
```

Reply:
or simply

config.Configure.Launcher.MemoryLimit = !clusterConf.DisableLimits

Reply:
I swear this was tested... but you are right, this doesn't do anything. I must have broken something when I was doing some quick adjustment. will fix, sorry.